src/stp/new_plays/DefendShot.cpp

```diff
@@ -118,13 +113,15 @@ void DefendShot::calculateInfoForDefenders() noexcept {
     stpInfos["defender_5"].setBlockDistance(HALFWAY);
 
     // When the ball moves, one defender tries to intercept the ball
+    auto closestBotUs = world->getWorld()->getRobotClosestToBall(world_new::us);
+    auto closestBotThem = world->getWorld()->getRobotClosestToBall(world_new::them);
```

Comment:
If  we do it like this anyway why not just have a getOurClosestRobotToBall() and getTheir...()